test/Test/Toml/Gen.hs

```diff
@@ -53,18 +58,15 @@ genVal = Gen.int (Range.constant 0 256)
 -- TODO: Arrays and Date.
 -- | Generates random value of 'AnyValue' type.
 genAnyValue :: MonadGen m => m AnyValue
-genAnyValue = do
-  let genBool = Gen.bool
-  let genInteger = toInteger <$> Gen.int (Range.constantBounded @Int)
-  let genDouble = Gen.double $ Range.constant @Double (-1000000.0) 1000000.0
-  let genText = Gen.text (Range.constant 0 256) Gen.alphaNum
-  Gen.choice
-      [ AnyValue . Bool    <$> genBool
-      , AnyValue . Integer <$> genInteger
-      , AnyValue . Double  <$> genDouble
-      , AnyValue . Text    <$> genText
-      , AnyValue . Date    <$> genDate
-      ]
+genAnyValue = Gen.choice
+    [ AnyValue . Bool    <$> genBool
+    , AnyValue . Integer <$> genInteger
+    , AnyValue . Double  <$> genDouble
+    , AnyValue . Text    <$> genText
+    , AnyValue . Date    <$> genDate
+    , AnyValue <$> genArray
+    , AnyValue <$> genArrayList
```

Comment:
`genArrayList` shouldn't be needed here actually. `genArray` generator should be able to generate either arrays of non-arrays (like array of integers) or arrays of arrays. `genArray` generator should be implemented in a such way that it produces any possible valid `Array`.